src/Symfony/Bundle/FrameworkBundle/Controller/AbstractController.php

```diff
@@ -241,20 +241,30 @@ protected function denyAccessUnlessGranted($attributes, $subject = null, string
      * Returns a rendered view.
      */
     protected function renderView(string $view, array $parameters = []): string
+    {
+        @trigger_error('Since symfony/framework-bundle 5.1: "renderView()" method is deprecated. Use "renderTemplate()" instead.', E_USER_DEPRECATED);
+
+        return $this->renderTemplate($view, $parameters);
+    }
+
+    /**
+     * Returns a rendered template.
+     */
+    protected function renderTemplate(string $templatePath, array $parameters = []): string
```

Comment:
should be named `$templateName`, not `$templatePath`. That's not a path.

Reply:
My reasoning was:

```
blog/index.html.twig

* Template name: 'index.html.twig'
* Template path: 'blog/index.html.twig'
* Template file path: '<project>/templates/blog/index.html.twig'
```

Reply:
that's the wrong expectation. What Twig calls the template name is that `blog/index.html.twig` (or `@App/index.hml.twig`). Twig never deals with `index.html.twig` here.

And there is nothing forcing a template name to look like a path (the FilesystemLoader is the one bringing such behavior, but namespaces already causes a difference here).